packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -32,12 +38,18 @@ export function AuthenticationBanner(): ReactElement {
         srcSet={`${laptopBg} 1440w, ${desktopBg} 1920w, ${bg} 2880w`}
         sizes="(max-width: 1440px) 100vw, (max-width: 1920px) 1920px, 100vw"
       />
-      <Section className="w-[32.5rem]">
-        <OnboardingHeadline
-          className={{ title: 'typo-mega3', description: 'mb-8 typo-title3' }}
-        />
+      <Section className="w-[32.5rem] gap-4">
+        {children}
+        {!children && (
+          <OnboardingHeadline
+            className={{
+              title: 'typo-mega3',
+              description: 'mb-8 typo-title3',
+            }}
+          />
+        )}
       </Section>
-      <Section className="w-[23.25rem] pt-2">
+      <Section className="w-[23.25rem]">
```

Comment:
Was the change necessary here?

Reply:
Removing it lines up the icon/avatar relative to the input according to the design. Moves the text up a little when on the fallback one, but still looks correct to me. Could conditionally add the padding if no children.